src/page-building/career/culture.tsx

```diff
@@ -0,0 +1,95 @@
+import React from 'react';
+import styled from 'styled-components';
+import { up } from '../../components/breakpoint/breakpoint';
+import { Teaser } from '../../components/teasers/teaser';
+import { Illustration } from '../../components/illustration/illustration';
+import { IllustrationType } from '../../components/illustration/illustration-set';
+import { SectionHeader } from '../../new-components/section-header/section-header';
+
+interface CultureAspect {
+  illustration: IllustrationType;
+  title: string;
+  description: string;
+}
+
+const TeaserGrid = styled.div`
+  display: grid;
+  gap: 24px;
+
+  justify-items: stretch;
+  grid-template-columns: repeat(auto-fit, 250px);
+  ${up('md')} {
+    gap: 70px;
+    grid-template-columns: 1fr 1fr 1fr;
+  }
+`;
+
+const ASPECTS: CultureAspect[] = [
```

Comment:
hard coded is fine because this component is local and not meant to be reused anywhere. That's why I removed the  props.